modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/dto/mdf-claim/getDTOFromMDFClaim.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import MDFClaimDTO from '../../../interfaces/dto/mdfClaimDTO';
+import MDFClaim from '../../../interfaces/mdfClaim';
+
+export function getDTOFromMDFClaim(mdfClaim: MDFClaim): MDFClaimDTO & MDFClaim {
```

Comment:
```suggestion
export function getDTOFromMDFClaim(mdfClaim: MDFClaim): MDFClaimDTO {
```

A funcao nao eh pra retornar o DTO? a tipagem tem que seguir a logica.